Testing/Modules/Security/CorsTests.cs

```diff
@@ -42,6 +42,28 @@ public async Task TestPermissive()
 
             Assert.AreEqual("*", response.GetHeader("Access-Control-Allow-Origin"));
 
+            Assert.AreEqual("*", response.GetHeader("Access-Control-Allow-Methods"));
+            Assert.AreEqual("Authorization", response.GetHeader("Access-Control-Allow-Headers"));

```

Comment:
I think, this should be `*, Authorization`

Reply:
Yep, I've fixed it 🙂